### ISSUE TITLE
Cannot read property 'useTabs' of undefined

### ISSUE THREAD

**Issue by thezimmee** [jonlabelle/SublimeJsPrettier#89]
### Explain the Issue and Expected Behavior

Running sublime prettier with a config file that has `useTabs: true` throws the following error:

```
------------------
 JsPrettier ERROR 
------------------

Prettier reported the following error(s):

[error] stdin: TypeError: Cannot read property 'useTabs' of undefined
[error]     at makeAlign (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:7657:17)
[error]     at makeAlign (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:7658:11)
[error]     at fits (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:7700:22)
[error]     at printDocToString$2 (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:7814:33)
[error]     at formatWithCursor (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:27649:26)
[error]     at Object.formatWithCursor (/Volumes/Home/Projects/cs-office2/node_modules/prettier/index.js:27907:12)
[error]     at format (/Volumes/Home/Projects/cs-office2/node_modules/prettier/bin/prettier.js:20364:21)
[error]     at thirdParty$1.getStream.then.input (/Volumes/Home/Projects/cs-office2/node_modules/prettier/bin/prettier.js:20467:19)
[error]     at <anonymous>
[error]     at process._tickCallback (internal/process/next_tick.js:188:7)

Process finished with exit code 2
```

My config file is `.prettierrc` and has the following (I've also tried formatting as YAML, JSON, and JS as `.prettierrc.js`):

```
printWidth: 120
singleQuote: true
semi: true
useTabs: true
```

### Prettier version

```
1.9
```

### JsPrettier Plug-in Version

```
1.17.1
```

### Platform Details

    - Sublime Text Version: Dev Channel
    - Sublime Text Build: 3156
    - Operating System Name: MacOS
    - Operating System Version: macOS Sierra 10.12.6

### Generated Prettier command line arguments

```
/Volumes/Home/Projects/cs-office2/node_modules/.bin/prettier --stdin --tab-width 4 --use-tabs true --stdin-filepath /Volumes/Home/Projects/cs-office2/Gruntfile.js --config-precedence prefer-file --config /Volumes/Home/.prettierrc
```

### The contents of your `User/JsPrettier.sublime-settings` file

```
{
	"debug": true,
	"auto_format_on_save": true,
	"auto_format_on_save_excludes": ["*/node_modules/*", "*/package.json", "/*.md", "/.*rc"],
	"allow_inline_formatting": false,
	"additional_cli_args": {
		"--config": "/Volumes/Home/.prettierrc",
		"--config-precedence": "prefer-file",
	}
}
```
    
### The contents of your `<project_name>.sublime-project` file (if applicable)

```
{
	"folders":
	[
		{
			"folder_exclude_patterns":
			[
				"_gsdata_",
				"node_modules",
				"build",
				"src2",
			],
			"path": "~/Projects/cs-office2/"
		}
	]
}
```

### Steps to reproduce the behavior

The steps one would take to reproduce and observe the problem.

1. Add `useTabs: true` to prettier configuration file (tried `.prettierrc`, `.prettierrc.js` in YAML, JSON, and JS).
2. Save file or select the `JsPrettier: Format Code` item from the command palette.
3. Error above is thrown.

**Comment by hudochenkov** [comment]
Fixed in Prettier 1.9.1 https://github.com/prettier/prettier/issues/3399

**Comment by thezimmee** [comment]
Confirmed the fix works. Thank you!
